var/spack/repos/builtin/packages/vtkm/package.py

```diff
@@ -17,39 +17,77 @@ class Vtkm(Package):
     architectures."""
 
     homepage = "https://m.vtk.org/"
-    url      = "https://gitlab.kitware.com/api/v4/projects/vtk%2Fvtk-m/repository/archive.tar.gz?sha=v1.1.0"
+    url      = "https://gitlab.kitware.com/api/v4/projects/vtk%2Fvtk-m/repository/archive.tar.gz?sha=v1.3.0"
     git      = "https://gitlab.kitware.com/vtk/vtk-m.git"
 
     version('master', branch='master')
+    version('1.3.0', "d9f6e274dec2ea01273cccaba356d23ca88c5a25")
+    version('1.2.0', "3295fed86012226c107e1f2605ca7cc583586b63")
     version('1.1.0', "6aab1c0885f6ffaaffcf07930873d0df")
 
     variant("cuda", default=False, description="build cuda support")
+    variant("logging", default=False, description="build logging support")
+    variant("mpi", default=False, description="build mpi support")
```

Comment:
This should probably default to True to match every other Spack package.

Reply:
Done.